test/functional/apps/visualize/_tag_cloud.js

```diff
@@ -61,7 +61,7 @@ export default function ({ getService, getPageObjects }) {
         expect(spyToggleExists).to.be(true);
       });
 
-      it('should show correct tag cloud data', async function () {
+      it.skip('should show correct tag cloud data', async function () {
```

Comment:
Should this be part of this PR?

Reply:
It was flaking out on me so I had to skip it. I recorded this in an issue here: https://github.com/elastic/kibana/issues/16383. The alternative would be to do this in master so we get a specific commit for this change, but I don't think that's necessary. What do you think?

Reply:
I'm OK with piggybacking it.  Small change.